testsuite/tests/typing-misc/external_arity.ml

```diff
@@ -19,7 +19,7 @@ external cmp : 'a -> 'b = "%compare"
 Line 1, characters 0-36:
 1 | external cmp : 'a -> 'b = "%compare"
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Error: Wrong arity for builtin primitive "%compare"
+Error: Wrong arity for builtin primitive %compare
```

Comment:
Mixed feelings about this particular change. I liked to see a string literal just as in the source. But then if we keep a string literal *and* quote it as code on dumb terminals this may get very heavy very fast. So maybe let's look back at this example with visible code markup.